guide/popular-topics/common-questions.md

```diff
@@ -33,6 +33,37 @@ const user = <message>.mentions.users.first();
 
 </branch>
 
+### How do I unban a user?
+
+<branch version="11.x">
+
+<!-- eslint-skip -->
+
+```js
+const id = args[0];
+<guild>.unban(id);
+```
+
+::: warning
+Because we cannot ping a user who isn't in the server, we have to pass in the user id. To do this, we use arguments, represented by `args` (see "Commands with user input").
```

Comment:
'(see "Commands with user input")' this should be hyperlinked (`[text](url)`), then ot can be more easily accessed